bin/gui-terminal

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+python3 ../ganga/GangaGUI/gui/webterm.py </dev/null &>/dev/null &
+sleep 15
+python3 ../ganga/GangaGUI/gui/autotype.py
```

Comment:
This can just be avoided, see suggested changes elsewhere.

Reply:
autotype.py is used to type start ganga gui without user interaction from the web terminal, i was going to use selenium but it 
suffers with browser issues if drivers are not installed

Reply:
I tried directly to input via javascript but this caused terminal to crash